common/components/componentsBySection/FindProjects/ProjectSearchBar.jsx

```diff
@@ -26,20 +26,15 @@ class ProjectSearchBar extends React.Component<{||}, State> {
 
   render(): React$Node {
     return (
-      <div className="ProjectSearchBar-root col-12">
+      <div className="ProjectSearchBar-root">
         <i className={GlyphStyles.Search}></i>
         <input
           className="ProjectSearchBar-input"
           onChange={e => this.setState({keyword: e.target.value})}
           onKeyPress={this._handleKeyPress.bind(this)}
-          placeholder="Enter keywords"
+          placeholder="Search by issue areas, roles, and more..."
```

Comment:
Did this text come from Design?  Because searching in the text search specifically does NOT search issue areas/roles/etc, just project titles and descriptions.

Reply:
We just talked about this in standup but yes, this came from design and I'll change the text to more accurately reflect the search's actual search.